### ISSUE TITLE
Error detected while processing function enfocado#highlighter:

### ISSUE THREAD

**Issue by aspiiire** [wuelnerdotexe/vim-enfocado#13]
I am using Neovim, using Plug, I have installed many other themes, this one after setting the theme with colorscheme enfocado it gives me the following error after I start nvim

```
Error detected while processing function enfocado#highlighter:                                                                                          
line    1:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
W18: Invalid character in group name                                                                                                                    
W18: Invalid character in group name                                                                                                                    
W18: Invalid character in group name                                                                                                                    
W18: Invalid character in group name                                                                                                                    
W18: Invalid character in group name                                                                                                                    
Error detected while processing /Users/abc/.config/nvim/plugged/vim-enfocado/colors/enfocado.vim:                                                      
line 1001:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
line 1002:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
line 1003:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
line 1004:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
line 1005:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
line 1006:                                                                                                                                              
W18: Invalid character in group name                                                                                                                    
line 1007:
W18: Invalid character in group name                                                                  
line 1008:                                      
W18: Invalid character in group name
line 1009:id character in group name 
```

**Comment by wuelnerdotexe** [comment]
Hello! @aspiiire: This problem seems to stem from the 207bfca903120d1dcbe29999728ce747cc1bce59 commit and is because [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) introduced a [BREAKING CHANGE](https://github.com/nvim-treesitter/nvim-treesitter/commit/42ab95d5e11f247c6f0c8f5181b02e816caa4a4f) and now uses new groups for highlighting and deprecated the old ones, but it seems that these new highlights just works as of version `nvim-0.8`, and it is strongly recommended that you upgrade to the `nightly` version if you use [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter).

Anyway, I will release a hotfix in the next few hours to prevent this issue in older versions. Thanks for using **Enfocado** and reporting this issue. 👁

**Comment by wuelnerdotexe** [comment]
The proposed fix (c6bafd8064f54f25416be2cb3b535075cd90f714) is already in the main branch, feel free to update and proceed to close this issue only if it now works as expected :)

**Comment by aspiiire** [comment]
> The proposed fix ([c6bafd8](https://github.com/wuelnerdotexe/vim-enfocado/commit/c6bafd8064f54f25416be2cb3b535075cd90f714)) is already in the main branch, feel free to update and proceed to close this issue only if it now works as expected :)

Thank you so much @wuelnerdotexe sorry I had a lot of things going on and forgot about this topic, as soon as I have a little bit of time If God wills I'm going to update it and let you know, thanks for your help! Amazing theme

**Comment by wuelnerdotexe** [comment]
@aspiiire thanks to you for choosing Enfocado \0/
